### DISCUSSION TITLE
Repeating a search query fails to provide results

### DISCUSSION THREAD

**Discussion by RyGuy101** [requarks/wiki#3753] [Error / Bug Report]
**Version:** 2.5.191

**Actual Behavior:** When entering a search query in the search box, clearing the the search box, and retyping the same query, the user sees "No pages matching your query," even if the query showed results the first time. Some further details:
* This is easy to replicate on https://docs.requarks.io using various search queries, e.g. "introduction"
* The issue doesn't reproduce 100% of the time, but it does most of the time.
* The issue persists until the page is refreshed.
* The issue has existed for at least the last few versions.
* The issue exists for both the Basic and PostgreSQL database searches.

**Expected Behavior:** The user sees the same results for the same search query.

There is a related issue where seemingly random search results come back when clearing a search query by backspacing.

Edit: This appears to be a frontend issue (see comments below).

**Pics:**
Normal behavior:
![image](https://user-images.githubusercontent.com/1429867/112281005-d38e9880-8c42-11eb-89c0-3f834c930fe4.png)
No results after repeated search:
![image](https://user-images.githubusercontent.com/1429867/112281711-970f6c80-8c43-11eb-8164-5438d1e94c48.png)
Random results after backspacing:
![image](https://user-images.githubusercontent.com/1429867/112281143-f8830b80-8c42-11eb-9f8f-dee0c1739dc8.png)

**Comment by nrayburn-tech** [comment]
Can confirm the issue on version 2.5.201 as well.

I can also confirm that the response from the graphql query does return the same results, so the backend is fine.  It is a frontend related issue.

**Reply by RyGuy101** [reply]
Thanks (also to ivandemec) for looking into it. I probably won't be looking at this again but we can hope someone will come along and spend the time digging through the frontend code.

**Comment by dtack98** [comment]
I can also confirm this an issue as well.

**Comment by ivandemec** [comment]
Here also. The graphql query returns same results. Frontend is the problem.
